portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -29,20 +29,35 @@
 
 /** Servlet that allows users to add comments and returns comment history  */
 @WebServlet("/comments")
-public class DataServlet extends HttpServlet {
-    
+public class DataServlet extends HttpServlet {  
+  
+  private static int maxComments;
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
     Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
 
+    String maxCommentsString = request.getParameter("numcomments");
```

Comment:
We might as well align the naming of the query parameter and what we're calling it here in the servlet. Personally, I like the "max comments" phrasing.

Reply:
That's a good idea, will change this!

Reply:
I was just going over this. By the query parameter did you mean the Query("Comment") part? Since this parameter is the entity kind it has to match the same naming as when the comments are added to datastore, so I'd keep the current phrasing. Or were you referring to the request.getParamater("numcomments") so that it matches the maxComments phrasing?

Reply:
Oops, I mean the parameter in the HTTP query string: `numcomments`.

Reply:
Done! :)